### PR TITLE
repositories: add vulkano-edge repo

### PR DESCRIPTION
**Pull request by thequickfixer**
add vulkano-edge repository (not sure if this is the right place to merge)

repo link: https://github.com/thequickfixer/vulkano-edge
bugzilla is linked double check in repositories.xml in the repo link to verify

### CODE REVIEW

**Review by jsmolic — CHANGES_REQUESTED** [review]
Please keep the list sorted

**Comment by thequickfixer** [comment]
> Please keep the list sorted

Okay, one momento just new to this that's all sorry about that.

**Comment by jsmolic** [comment]
Thanks, can you squash the commits?

**Comment by thequickfixer** [comment]
> Thanks, can you squash the commits?

How do I squash a commit, this is kind of my first time using github. I hope it's not a problem asking. I've been doing this via github online.

**Comment by thequickfixer** [comment]
> Thanks, can you squash the commits?

I'll just delete the repo and create a new one and commit once? would that be advisable?

**Comment by jsmolic** [comment]
@thequickfixer No need to delete the repo. You can fix everything in a local clone of the repository, however I suggest doing changes outside of the master branch, you might need to create a new PR for that since this one is from your master branch.

**Comment by thequickfixer** [comment]
> > > Thanks, can you squash the commits?
> > 
> > 
> > How do I squash a commit, this is kind of my first time using github. I hope it's not a problem asking. I've been doing this via github online / offline for manifests, I know how to do everything locally. Just not able to squash the commits online via github from what I can see.
> 
> You can squash locally using `git rebase -i`, for example `git-rebase -i HEAD~2` (to change the 2 latest commits), change `pick` to `s` on the last commit and use `git push --force origin <branch_name>` to force push changes

Alright I'll try that and let you know what the result of it is

**Comment by thequickfixer** [comment]
> > > Thanks, can you squash the commits?
> > 
> > 
> > How do I squash a commit, this is kind of my first time using github. I hope it's not a problem asking. I've been doing this via github online / offline for manifests, I know how to do everything locally. Just not able to squash the commits online via github from what I can see.
> 
> You can squash locally using `git rebase -i`, for example `git-rebase -i HEAD~2` (to change the 2 latest commits), change `pick` to `s` on the last commit and use `git push --force origin <branch_name>` to force push changes

thank you, I think it's done

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -4666,6 +4666,16 @@
...
     <feed>https://cafarelli.fr/cgi-bin/cgit.cgi/voyageur-overlay/atom</feed>
   </repo>
+  <repo quality="experimental" status="unofficial">
+    <name>vulkano-edge</name>
+    <description lang="en">A repository just for Vulkan stuff for Gentoo</description>
+    <homepage>https://github.com/thequickfixer/vulkano-edge</homepage>
+    <owner type="person">
+      <email>TeaFor2Heavy@protonmail.com</email>
+    </owner>
+    <source type="git">git://github.com/thequickfixer/vulkano-edge.git</source>
```
Comment (jsmolic) [inline files/overlays/repositories.xml:4676]: I just saw this now, git:// protocol is not supported on GH anymore (https://github.blog/2021-09-01-improving-git-protocol-security-github/), let's change it to https://

Comment (thequickfixer) [reply files/overlays/repositories.xml:4676]: okay, will change and do more rebasing lol...

Comment (thequickfixer) [reply files/overlays/repositories.xml]: going to rebase one moment

Comment (jsmolic) [reply files/overlays/repositories.xml]: Sure, FYI you don't need to push every time you rebase, make sure everything is done and clean in your local repo and push after you verify it ;-)

Comment (thequickfixer) [reply files/overlays/repositories.xml]: rebased

**Comment by thequickfixer** [comment]
well that was interesting... thank's jsmolic!

File: files/overlays/repositories.xml (hunk 1 above)
Comment (thequickfixer) [reply files/overlays/repositories.xml]: your a big help!

**Review by jsmolic — APPROVED** [review]
Thanks

**Pull request closed without merging**